packages/central-server/src/apiV2/DownloadFiles.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import {
+  S3,
+  S3Client,
+  ValidationError,
+  respondWithDownload,
+  writeStreamToFile,
+} from '@tupaia/utils';
+import { RouteHandler } from './RouteHandler';
+import { getTempDirectory } from '../utilities';
+import { zipMultipleFiles } from './utilities';
+
+export class DownloadFiles extends RouteHandler {
```

Comment:
Couldn't quite decide about whether to call this `DownloadFiles` or perhaps another name like `DownloadResources`.

`DownloadFiles` feels a bit generic (for instance we already have the `export/download` route), but there isn't really a term in our system to describe this (and I don't think there needs to be for now).